tests/kolainst/destructive/filelists

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
```

Comment:
@cgwalters @jmarrero Do you guys have any input of how thorough this test should be? I've currently implemented it such that tests almost every case by repeatedly installing packages and checking how the filelists change. However, installing packages take an extremely long time and it timed out on me. I know we can increase the timeout, but I don't think it's realistic to have a test that takes forever to finish.

Reply:
I think most of test run in parallel so it is very likely this won't add time to our tests ,but lets let it run and see how it looks. For the tests, I would just remove anything using copr as those can go away. I think even without those we are testing most of the code paths.

Reply:
With regards to tests using `rpmostree_busctl_call_os` such as `WhatProvides`, `GetPackages`, and `Search`, do they have to be run offline, as seen in `misc.sh`? I've been trying to implement these into my kola test with internet enabled so I can check to see if filelists are being installed, but I always seem to get `Call failed: Connection timed out`. Any input on this?

Reply:
I find that even as I'm running dbus manually:
```
stateroot=$(rpm-ostree status --booted --json | jq -r '.deployments[0].osname')
  ospath=/org/projectatomic/rpmostree1/${stateroot//-/_}
  busctl call org.projectatomic.rpmostree1 $ospath \
    org.projectatomic.rpmostree1.OS WhatProvides as 1 test
```
On a build of FCOS, I also get `Call failed: Connection timed out`. However, when I call it a second time immediately after, it seems to work.

Reply:
Fixed by setting the timeout flag to 60 seconds for busctl